pkg/install/install_gemini_commands.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+const costTemplate = `description="Answer natural language questions about GKE-related costs by leveraging the bundled cost context instructions within the gke-mcp server."
+prompt = """
+You are a GKE cost and optimization expert. Answer the user's question about GKE costs, optimization, or billing using the comprehensive cost context available in the GKE MCP server.
+
+User Question: {{args}}
+
+Based on the GKE cost context available, provide a detailed and helpful response that includes:
+
+1. **Direct Answer**: Address the specific cost question or optimization request
+2. **BigQuery Integration**: Explain how to use BigQuery for cost analysis if relevant
+3. **Cost Allocation**: Mention GKE Cost Allocation requirements when applicable
+4. **Actionable Steps**: Provide concrete next steps or commands when possible
+5. **Resource References**: Point to relevant GCP documentation or console links
+
+Key points to remember:
+- GKE costs come from GCP Billing Detailed BigQuery Export
+- BigQuery CLI (bq) is preferred over BigQuery Studio when available
+- GKE Cost Allocation must be enabled for namespace and workload-level cost data
+- Required parameters include BigQuery table path, time frame, project ID, cluster details
+- Use the cost analysis queries from the GKE MCP documentation as templates
+
+Always be helpful, specific, and actionable in your response.
+"""`
+
+// getCostTemplate retrieves the cost template content
+func getCostTemplate() []byte {
+	return []byte(costTemplate)
```

Comment:
Instead of embedding markdown into a Go file, how about having a list of .md files under install/commands, one for each command. 

For example, this content could go in install/commands/costs.md. Then, in installCustomCommands you can just iterate over the files and create one command for each file. 

I see two main benefits from this a) the markdown is externalized in an md file that makes it easier to edit/visualize/review and b) we can add more commands just by adding md files to the directory.